docs/website/plugins/nikita-packages/gatsby-node.js

```diff
@@ -0,0 +1,101 @@
+
+const crypto = require("crypto")
+const path = require("path")
+const fs = require('fs')
+
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createTypes } = actions
+  createTypes(`
+    """
+    NikitaPackage Node
+    """
+    type NikitaPackage implements Node @infer {
+      name: String!
+      shortName: String!
+      slug: String!
+      version: String
+      edit_url: String!
+      frontmatter: Frontmatter
+    }
+    type Frontmatter {
+      title: String!
+      navtitle: String!
+      description: String
+      keywords: [String]
+    }
+  `)
+}
+
+exports.sourceNodes = async ({ actions, createNodeId }) => {
+  const { createNode } = actions
+  // Get all packages
+  const nikitaDir = `${__dirname}/../../../../packages`
+  fs.readdirSync(nikitaDir)
+    .filter(file => fs.statSync(path.join(nikitaDir, file)).isDirectory() && file != 'nikita')
```

Comment:
oh no, you are blocking the all gatsby generation process

Reply:
not for a long, ok